libiocage/lib/helpers.py

```diff
@@ -102,6 +102,26 @@ def validate_name(name):
     return bool(validate.fullmatch(name))
 
 
+# data -> bool | default
+# This little helper function will try to parse booleans from strings
+# On success, it returns the parsed boolean, on failure it returns the
+# `default`. By default, `default` is `False`.
+def parse_bool(data, default=False):
```

Comment:
so, i was playing with this, because there were a bunch of different use-cases:
but most of the time, the comparison was directly against a `Truthy` value, which meant to me, that in absence of truth, false would've been the _default_
However, a number of other cases use it in such a away that the value is essentially passed-thru, but modified in case it can be interpreted as `Truthy` or `Falsey`.